### DISCUSSION TITLE
umi创建的项目有很多ts错误，怎么消除，很烦人

### DISCUSSION THREAD

**Discussion by qinhuaihe** [umijs/umi#10073] [Q&A]
![WeChat Screenshot_20221215115739](https://user-images.githubusercontent.com/18019463/207769239-d24a65f7-31c3-4beb-89ee-d4c123b51dc7.png)

![WeChat Screenshot_20221215120039](https://user-images.githubusercontent.com/18019463/207769672-a934d6df-deed-4f4c-95b9-4611654af44f.png)

umi4的react-router不是升级到6了吗，看起来有些依赖版本不正确

**Comment by fz6m** [comment]
1. 升级 `umi` 到 [最新](https://www.npmjs.com/package/umi?activeTab=versions) 的版本

2. 启动项目 `pnpm dev`

3. reload 编辑器，ctrl + shift + p 输入 reload

**Reply by qinhuaihe** [reply]
umi就是最新的，这个项目也是新创建的
